packages/backend/src/plugins/search.ts

```diff
@@ -73,7 +73,10 @@ export default async function createPlugin({
 
   indexBuilder.addCollator({
     defaultRefreshIntervalSeconds: 600,
-    collator: new DefaultTechDocsCollator({ discovery, logger }),
+    collator: DefaultTechDocsCollator.fromConfig(config, {
```

Comment:
Does the previous way still work or should we be updating `create-app` and creating a migration note saying how to update?

Reply:
The old way still works, it uses the constructor directly. There is a new optional parameter in the newer version of the constructor, `legacyPathCasing`, which defaults to false. For now it is fine, since TechDocsCollator/Search is not yet officially 'released' and is missing from create-app altogether.